tools/DynamoPackagesAnalyzer/Models/Greg/PackageVersion.cs

```diff
@@ -0,0 +1,50 @@
+namespace DynamoPackagesAnalyzer.Models.Greg
+{
+    /// <summary>
+    /// Greg Client dto
+    /// </summary>
+    internal class PackageVersion
```

Comment:
so does this class need to be in sync with whatever Greg represents a package version ?
Same question for PackageHeader.

Reply:
Yes, in fact those are the same dto's used by Greg